src/client/app/translations/data.json

```diff
@@ -146,6 +151,8 @@
 		"map.new.submit": "Save and continue",
 		"map.note": "Note",
 		"map.upload.new.file": "Redo",
+		"map.load.complete": "Map load complete from",
```

Comment:
Slightly wrong line for alphabetical.

Reply:
fixed this change, also moved a few other lines